other/materials_designer/create_nanowire.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>by creating a slab (with specific Miller indices)and cutting a cylinder along its Z axis, then, optionally, rotating to align along X axis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>